news/_posts/translation/2020-09-09-do-you-speak-prestashop-august-2020.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – August 2020 edition"
+subtitle: "Where you'll learn that 3 new languages projects have opened"
+date:  2020-09-09 12:00:00
+authors: [JulieVarisellaz]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in August.
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+### New manager of the translation projects 
+
+My name is Julie. I joined PrestaShop on September 1st as the new Product Content Manager. I am very happy to start this new adventure and I am looking forward to working together on this translation project! :fire:
+ 
+
+### New projects have been opened!
+
+Would you like to try a new challenge? If you are fluent in one of the languages below, don’t hesitate to contribute. Here are the new projects that have been opened in August:
+Spanish (Costa Rica), French (Belgium), and Maltese.
```

Comment:
I dont know if you're aware but going to the next line in Markdown does not work. You need to have an empty line between 2 statements to have a real "going to the next line" 😉 

You can preview how your article will be displayed by going on https://github.com/PrestaShop/prestashop.github.io/pull/770/files and clicking on the menu (three dots) on top right and then "View file"

Reply:
@Julievrz 
Please have a look at this:
- https://daringfireball.net/projects/markdown/syntax

@matks actually, there is a way to force a next line in markdown: just add 2 spaces at the end of the line. I'll make a suggestion.